src/ch00-06-constructor.md

```diff
@@ -0,0 +1,15 @@
+# Constructor
+
+Constructors are a special type of function that runs only once when deploying a contract, and can be used to initialize the state of the contract.
+
+Some important rules to note:
+
+- Your contract can't have more than one constructor.
+- Your constructor function must be named constructor.
+- Lastly, it must be annotated with the #[constructor] attribute.
+
+Here's a simple example that demonstrates the use of constructor:
```

Comment:
> Here's a simple example that demonstrates the use of constructor:

Here's a simple example that demonstrates how to initialize the state of a contract on deployment by defining a constructor.